conditional/evaluations/views.py

```diff
@@ -1,4 +1,4 @@
-from evaluations.models import SpringEval, SpringEvalSerializer
+from evaluations.models import SpringEval, SpringEvalSerializer # pylint: disable=E0611
```

Comment:
This is a workaround. I couldn't get generated-members to cover this use case. I think https://github.com/PyCQA/pylint/pull/3634 will solve it, but it's not merged yet.